app-modules/care-team/database/factories/CareTeamFactory.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Assist\CareTeam\Database\Factories;
+
+use Illuminate\Database\Eloquent\Factories\Factory;
+
+/**
+ * @extends \Illuminate\Database\Eloquent\Factories\Factory<\Assist\CareTeam\Models\CareTeam>
+ */
+class CareTeamFactory extends Factory
+{
+    /**
+     * Define the model's default state.
+     *
+     * @return array<string, mixed>
+     */
+    public function definition(): array
```

Comment:
Can we add some factory data here?

Reply:
What would I put here? Right now they're just a pure pivot with no extra data like Subscriptions. I could delete the factory like Subscriptions?

Reply:
@mxm1070 That's fine as well since we are not using it for now.